_data/header_navmenu_datafile.json

```diff
@@ -328,6 +328,13 @@
                             "URL": "https://github.com/opensearch-project",
                             "ClassValue" : "drawer-content-subheader-link",
                             "Icon": "OpenSearch-Github-condensedImage"
+                        },
+                        {
+                            "Name": "Project Roadmap",
+                            "SubText": "What is next for the OpenSearch project?",
+                            "URL": "https://github.com/orgs/opensearch-project/projects/220",
```

Comment:
Hi @kaimmej the new roadmap is this: https://github.com/orgs/opensearch-project/projects/206
```suggestion
                            "URL": "https://github.com/orgs/opensearch-project/projects/206",
```

cc: @Pallavi-AWS @getsaurabh02 

Thanks!

Reply:
We are moving towards new dashboard: https://github.com/orgs/opensearch-project/projects/206

Reply:
Thank you for the feedback! 
### Updates:
- I have updated the project roadmap link to point to the correct project
- Additionally, I added a card that distinguishes it from the other navbar items. 
- Roadmap moved to the docs dropdown. 

### Image
![image](https://github.com/user-attachments/assets/6c23d869-61a5-46ad-9a54-0223817ce6dc)

Reply:
this looks great @kaimmej